tf2onnx/tfonnx.py

```diff
@@ -769,7 +769,7 @@ def concatv2_op(ctx, node, name, args):
     # if any input is empty, remove the input and concat the others
     # NOTE: workaround for https://github.com/Microsoft/onnxruntime/issues/681
     for i, inp in enumerate(node.inputs):
-        if inp.is_const() and inp.get_tensor_value() == []:
+        if inp.is_const() and inp.get_tensor_value(as_list=False).size == 0:
```

Comment:
err, what if all inputs are const and empty?
we don't really need to support this case, but better to add a check and throw.